### DISCUSSION TITLE
Pekko feature set

### DISCUSSION THREAD

**Discussion by jausmann-wc** [apache/pekko#1106] [General]
Is it the mission of Pekko to contain only the previous and any future open source features of Akka? Or could entirely new features be added which are not part of any version of Akka?

**Comment by mdedetrich** [comment]
> Is it the mission of Pekko to contain only the previous and any future open source features of Akka? Or could entirely new features be added which are not part of any version of Akka?

Entirely new features are welcome as long as they are in the spirit of the project (if this is in doubt it should be discussed first).

This is already happening, as an example pekko streams 1.1.x has an entire set of new operators.

**Comment by He-Pin** [comment]
Pekko is open sourcing,  so features come in as community needs not any specific customer needs.
